src/sentry/static/sentry/app/views/settings/organizationSecurityAndPrivacy/organizationSecurityAndPrivacyContent.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import JsonForm from 'app/views/settings/components/forms/jsonForm';
+import Form from 'app/views/settings/components/forms/form';
+import AsyncView from 'app/views/asyncView';
+import {GetEndPointsOutput} from 'app/components/asyncComponent';
+import {Organization} from 'app/types';
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {updateOrganization} from 'app/actionCreators/organizations';
+import organizationSecurityAndPrivacy from 'app/data/forms/organizationSecurityAndPrivacy';
+
+import {OrganizationSecurityAndPrivacyProps} from './types';
+
+class OrganizationSecurityAndPrivacyContent extends AsyncView<
+  OrganizationSecurityAndPrivacyProps
+> {
+  static contextTypes = {
```

Comment:
These are both passed down as props from `<OrganizationSecurityAndPrivacy>`

Reply:
if I remove the contextTypes then the tests don't pass

Reply:
If that's the case, we should fix the tests. My assumption is that the tests are mounting `-Content` component and not the container (which the container provides the organization) and that when we are mounting the component under test, we are not providing an `organization` prop.